README.md

```diff
@@ -1,5 +1,10 @@
 # Bulletproofs
 
+<img
+ width="100%"
+ src="https://user-images.githubusercontent.com/698/46373713-9cc40280-c643-11e8-9bfe-2b0586e40369.png"
+ TODOsrc="https://doc.dalek.rs/assets/bulletproofs-diagram.png"/>
```

Comment:
I think this could just use the githubusercontent URL for now, and change the source to doc.dalek.rs later. The reason is that I need to figure out a better system for the `assets` directory for doc.dalek.rs anyways, and some scripts for doing deployment, etc., and that's more work that adding this image doesn't need to be coupled to.

Reply:
Works for me.